thunderbird/utils.py

```diff
@@ -66,28 +66,37 @@ def collect_output_files(varname, outdir=os.getcwd()):
     return [file for file in os.listdir(outdir) if varname in file]
 
 
-def build_meta_link(varname, desc, outfiles, outdir=os.getcwd()):
+def build_meta_link(
+    varname,
+    desc,
+    outfiles,
+    format_name="netCDF",
+    fmt=FORMATS.NETCDF,
```

Comment:
Not sure if this is possible but we may want to derive the `format_name` from the `fmt` somehow.

Reply:
Off the top of my head, the best thing I can think of is having a dictionary map the `fmt` to the `format_name`, but that's not really flexible since the `fmt` the user provides has to exist in said dictionary.

Reply:
That's fair, we can leave it as is.